netlify-tutorial.md

```diff
@@ -48,10 +50,10 @@ You can name the repository by adding `--fork-name <name>` to the command.
 
 Follow the instructions below to configure continuous deployment. This will automatically deploy your changes. Make sure to run the commands in the root directory of the repository.
```

Comment:
```suggestion
Follow the instructions below to configure continuous deployment. This will automatically deploy your changes.
```

Now that we have the `cd nextjs-netlify`, we no longer need this part.